src/elements.rs

```diff
@@ -107,6 +123,45 @@ impl Element {
 
 /// [Element State](https://www.w3.org/TR/webdriver1/#element-state)
 impl Element {
+    /// Return true if the element is currently selected.
+    ///
+    /// See [13.1 Is Element Selected](https://www.w3.org/TR/webdriver1/#is-element-selected)
+    /// of the WebDriver standard.
+    pub async fn is_selected(&mut self) -> Result<bool, error::CmdError> {
+        let cmd = WebDriverCommand::IsSelected(self.element.clone());
+        match self.client.issue(cmd).await? {
+            Json::Bool(v) => Ok(v),
+            Json::Null => Ok(false),
```

Comment:
Is this valid according to the spec? If not, we should leave a comment about why it's needed. And same for the other two instances below.

Reply:
Yes I believe it is valid. Which part were you unsure of? The inclusion of `IsSelected` ? or the boolean value?

Here's the spec: https://www.w3.org/TR/webdriver1/#is-element-selected
(which is already mentioned in the docs here)
```
Let selected be the value corresponding to the first matching statement:

element is an input element with a type attribute in the Checkbox- or Radio Button state

    The result of element’s checkedness. 
element is an option element

    The result of element’s selectedness. 
Otherwise
    False. 
```
Those are all boolean. And, this is tested in the `thirtyfour` doctest.

Reply:
Ah, sorry, no, I mean the `Json::Null`. Is it possible for this API to return a `null` at all? If it's not, we should return that as `NotW3C`.